comid/referencevalue.go

```diff
@@ -1,24 +1,86 @@
-// Copyright 2021 Contributors to the Veraison project.
+// Copyright 2021-2024 Contributors to the Veraison project.
 // SPDX-License-Identifier: Apache-2.0
 
 package comid
 
-import "fmt"
+import (
```

Comment:
The file name no longer match the contents of the file, earlier it was correctly done.

Now the name should be valuetriples.go